modules/lss-sumDailyMissions/sumDailyMissions.user.js

```diff
@@ -167,7 +167,7 @@
             extended: 'construida',
             extended2: 'Extendido',
             built: 'construido',
-            teame: ' Alianza',
+            teame: 'Hospital \\(alianza\\)',
```

Comment:
change it to \\(alianza\\)

Reply:
and change num_teame back to
Tratamiento de alianza suma / créditos
or
Suma / Créditos de Tratamiento de alianza 
because it counts both. Rescue service and police